Source/WebCore/css/CSSCounterStyleRegistry.cpp

```diff
@@ -137,13 +142,6 @@ RefPtr<CSSCounterStyle> CSSCounterStyleRegistry::resolvedCounterStyle(const Atom
 
 CounterStyleMap& CSSCounterStyleRegistry::userAgentCounterStyles()
 {
-    // FIXME: counter-style should get pre-defined counters from UA stylesheet rdar://103021161.
-    auto initialStyles = []() {
-        CounterStyleMap map;
-        map.add("decimal"_s, CSSCounterStyle::createCounterStyleDecimal());
```

Comment:
Can you delete this `CSSCounterStyle::createCounterStyleDecimal()` function? I assume it's unused now